src/libs/auth.js

```diff
@@ -52,13 +98,14 @@ const becameRegistered = (oldState, state) => {
  */
 export const ensureBillingScope = async () => {
   if (!hasBillingScope()) {
-    const options = new window.gapi.auth2.SigninOptionsBuilder({ scope: 'https://www.googleapis.com/auth/cloud-billing' })
-    await getAuthInstance().currentUser.get().grant(options)
-    // Wait 250ms before continuing to avoid errors due to delays in applying the new scope grant
-    await Utils.delay(250)
+    await signIn(true)
   }
 }
 
+const becameRegistered = (oldState, state) => {
```

Comment:
This just moved locations in the file?

Reply:
Ah yeah was just bunching the scope-related methods together